@xen-orchestra/lite/src/jobs/vm-start-on.job.ts

```diff
@@ -0,0 +1,36 @@
+import { hostArg, vmsArg } from '@/jobs/args'
+import { isVmOperationPending } from '@/libs/vm'
+import { VM_OPERATION, VM_POWER_STATE } from '@/libs/xen-api/xen-api.enums'
+import { useXenApiStore } from '@/stores/xen-api.store'
+import i18n from '@core/i18n'
+import { defineJob, JobError, JobRunningError } from '@core/packages/job'
+
+export const useVmStartOnJob = defineJob('vm.start-on', [vmsArg, hostArg], () => {
+  const xapi = useXenApiStore().getXapi()
+  const { t } = i18n.global
```

Comment:
Shouldn't it be `useI18n()` like in the other files ?
```suggestion
  const { t } = useI18n()
```